values-10k-to-1M.yaml

```diff
@@ -0,0 +1,32 @@
+# Minimal settings for scale of 10k to 1M events/month.
+# Note that this is experimental, please let us know how this worked for you.
+
+ingress:
+  # -- URL to address your PostHog installation. You will need to set up DNS after installation. GCP tutorial: https://cloud.google.com/dns/docs/tutorials/create-domain-tutorial
```

Comment:
I'd avoid linking to 3rd order documentation here. We already have:
- Docs in posthog.com
- README.md (incoming)
- Notes.txt

We should instead double down on having proper guides in 2 of the three.